sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 12.0.0b4 (Unreleased)
```

Comment:
Don't forget to put a date in here next week ;)
Or you could proactively put in Tuesdays date and amend it later if for any reason Tuesday doesn't happen....